pkg/plugins/manager/manager.go

```diff
@@ -228,11 +274,29 @@ func (pm *PluginManager) Apps() []*plugins.AppPlugin {
 	return rslt
 }
 
+func (pm *PluginManager) Panels() []*plugins.PanelPlugin {
+	pm.pluginsMu.RLock()
+	defer pm.pluginsMu.RUnlock()
+
+	var rslt []*plugins.PanelPlugin
```

Comment:
can use `copy` here

Reply:
`pm.panels` is a map so unfortunately `copy` can't give us any help in this case. However, I will pre-allocate those slices now that I see that